### DISCUSSION TITLE
3D printed case for Galagino - ESP32 Arcade Emulator

### DISCUSSION THREAD

**Discussion by Kiwi-Bushwalker** [harbaum/galagino#17] [Show and tell]
![Galagino (small)](https://github.com/harbaum/galagino/assets/63612585/fb3dfc30-1f1f-4d7d-b017-af258da4a7d2)

Build instructions and STL files etc on [Hackster.io](https://www.hackster.io/john-bradnam/galagino-esp32-arcade-355dde)

**Comment by harbaum** [comment]
That's really cool. A few days ago I also manufactured some slightly different PCBs to ease assembly. I wonder if your joystick is mechanically still compatible with my laser cut case.

**Reply by Kiwi-Bushwalker** [reply]
I already had the SMD 5-Way 10mm x 10mm x 9mm switches so the PCB was designed for it. I found through-hole equivalents on AliExpress (https://www.aliexpress.com/item/4000681560472.html). This means you could mount it in the same way that you used for the 6mm x 6mm tactile switches.

**Reply by harbaum** [reply]
I also have some of these and found they imho need to much force. Also the fact that they have a push function wasn't what I hoped for.

But I like to have the option to mix the case and joystick types. So the question was if someone building the laser cut case could just use your joystick with it. I think that would work, wouldn't it? It's still the exact same shape?

**Reply by Kiwi-Bushwalker** [reply]
Yes it is exactly the same shape. So it should fit into the laser cut case without any modification.
